src/SFA.DAS.Roatp.Functions.UnitTests/ApplicationExtractTests.cs

```diff
@@ -168,5 +168,21 @@ public async Task EnqueueApplyFilesForExtract_Enqueues_Requests()
 
             _applyFileExtractQueue.Verify(x => x.AddAsync(It.IsAny<ApplyFileExtractRequest>(), It.IsAny<CancellationToken>()), Times.AtLeastOnce);
         }
+
+        [Test]
+        public async Task ExtractOrganisationManagementForApplication_Saves_OrganisationManagement_Answers()
```

Comment:
It is worth adding a test for the conversion including time in role months calculation logic.

Reply:
Added Assert for MonthsinRole